meta/runtime.yml

```diff
@@ -952,7 +952,7 @@ plugin_routing:
           modules. This has never been supported or documented, and will stop working
           in community.general 9.0.0.
     database.saphana.hana_query:
-      redirect: community.general.hana_query
+      redirect: community.sap_libs.sap_hdbsql
```

Comment:
Please do not touch these entries, instead add new ones for the actual modules. These entries are for the old long-names of the modules.